benchmark/_interfaces.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Operations for the control service benchmarks.
+"""
+
+from zope.interface import Interface
+
+
+class IScenario(Interface):
+    """
+    A scenario in which an operation occurs.
+    """
+
+    def __init__(self, clock, control_service):
+        """
+        Provide the scenario with a clock and control service.
+        """
```

Comment:
Couple things about zope.interface
 - the convention is to leave the `self` argument out of the method signatures (since the caller isn't expected to pass this argument)
 - `__init__` is typically not part of the interface (ie, creation and initialization of the object that provides the interface is not part of the interface - though sometimes it's part of some other interface)